data/map planets.txt

```diff
@@ -2761,6 +2972,25 @@ planet Martini
 		threshold 8500
 		fleet "Large Republic" 22
 
+planet Maspa-Viir-Kella
+	attributes successor uninhabited
+	landscape land/clouds_03
+	description `This planet is about as large as a terrestrial planet can be before it is considered a gas giant. It is surrounded by an extremely thick atmosphere that is just barely within the tolerances of traditional shipbuilding techniques, and that buffets your vessel with high winds before you even make landing. It is hard to see anything at any distance on the ground; the fog is simply too thick.`
+	government Uninhabited
+
+planet Mavra-Sol-Kvel
+	attributes successor wealthy
+	landscape land/fields14
+	description `This green and well-populated moon enjoys a temperate climate and spectacular vistas, both of its own surface and of the gas giant it orbits. The gravity here is quite low, and the Successors you see here tower over you even more noticeably than on other worlds. There are several cities on the surface, most along the equator.`
+		to display
+			not "known to the successors"
+	description `Shielded from the persistent asteroid impacts of this system by its parent gas giant, Mavra-Sol-Kvel enjoys a temperate climate and spectacular vistas. The low gravity here is greatly appreciated by the Successors, and consequently the moon supports a substantial population despite its small size, most of whom live in several cities along the equator. It also allows for a number of towering mountain ranges, many containing cliffs and canyons several kilometers high that are frequently visited by tourists and thrill-seeker pilots alike.`
+		to display
+			has "known to the successors"
+	spaceport `The spaceport is large but efficient, and plastered in bright and friendly colors, much like the landscape surrounding it. Most of the buildings are organic in shape, arcing and bubbling from place to place like seafoam, and despite the fanciful design, they also appear to be servicing a large volume of travellers, merchants, and military personnel, with color-coded lines leading to different areas.`
```

Comment:
Not super gung-ho about this change, so feel free to resolve if you feel it doesn't fit.
```suggestion
	spaceport `The spaceport is large but efficient, plastered in bright and friendly colors much like the landscape surrounding it. Most of the buildings are organic in shape, arcing and bubbling from place to place like seafoam. Despite the fanciful design, they also appear to be servicing a large volume of travellers, merchants, and military personnel, with color-coded lines leading to different areas.`
```